### ISSUE TITLE
add support for host resource types

### ISSUE THREAD

**Issue by logicminds** [nwops/puppet-retrospec#30]

**Comment by logicminds** [comment]
```
Users/cosman/.rubygems/gems/puppet-retrospec-0.8.0/lib/retrospec.rb:161:in `generate_file_path': node retrospec does not support this resource type yet (RuntimeError)
        from /Users/cosman/.rubygems/gems/puppet-retrospec-0.8.0/lib/retrospec.rb:133:in `safe_create_resource_spec_files'
        from /Users/cosman/.rubygems/gems/puppet-retrospec-0.8.0/lib/retrospec.rb:77:in `block in create_files'
        from /Users/cosman/.rubygems/gems/puppet-retrospec-0.8.0/lib/retrospec.rb:76:in `each'
        from /Users/cosman/.rubygems/gems/puppet-retrospec-0.8.0/lib/retrospec.rb:76:in `create_files'
        from /Users/cosman/.rubygems/gems/puppet-retrospec-0.8.0/bin/retrospec:28:in `<top (required)>'
        from /Users/cosman/.rubygems/bin/retrospec:23:in `load'
        from /Users/cosman/.rubygems/bin/retrospec:23:in `<main>'

```

**Comment by logicminds** [comment]
this was released in 0.8.1
